setup-environment

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+python -m venv .venv
+source .venv/bin/activate
+pip install -r requirements.txt
+ansible-galaxy install -r requirements.yml
+
+echo "\n\nSetup complete. Activate the virtualenv 'source .venv/bin/activate' and then run the cli 'rop'"
```

Comment:
`bash`'s `echo` does not interpret escape chars by default. you need to either use `echo -e` (non POSIX, afaik) or `printf`